### ISSUE TITLE
Add support for Carthage and use of EZAudio as a dynamic framework

### ISSUE THREAD

**Issue by AndrewSB** [syedhali/EZAudio#210]

**Comment by imownbey** [comment]
Would love this

**Comment by AndrewSB** [comment]
@imownbey figured this out #211

**Comment by daniel-shin1104** [comment]
+1

**Comment by imownbey** [comment]
I am not totally sure that what you merged in actually works. There are still no header files I think there are a couple steps left to do

**Comment by AndrewSB** [comment]
@imownbey saw your fork and pulled those changes in to the PR https://github.com/AndrewSB/EZAudio/commit/5f283af9d322cf2f12a833f20582911745117ae8

**Comment by AndrewSB** [comment]
I think I have it working - just tried building with [this](https://github.com/AndrewSB/ZLSinusWaveView) library and it seemed to work

**Comment by syedhali** [comment]
Hey guys, to be honest I'm not too hip on Carthage at the moment. I saw #211, but doesn't it also need a target for the OSX framework as well?

**Comment by AndrewSB** [comment]
You are right @syedhali, it would need a target for OS X, I have not implemented it because I've never targeted OS X before.

I could try it, but I think someone with more experience on integrating Carthage into OS X projects should take a stab at it.
